build.sbt

```
@@ -201,55 +94,14 @@ lazy val docs = project
   .in(file("docs"))
   .dependsOn(coreJVM, catsJVM)
   .enablePlugins(MdocPlugin)
+  .enablePlugins(NoPublishPlugin)
```

I can do it in a follow-up PR, but I think if you add
```scala
addSbtPlugin("org.typelevel" % "sbt-typelevel-site" % "0.4.6")
```
we can get a website for free, since it looks like the docs are not currently published.
```suggestion
  .enablePlugins(TypelevelSitePlugin)
```

---

Also
```scala
ThisBuild / tlSitePublishBranch := Some("master")
```

---

Oh, and `intro.md` should be renamed to `index.md` :)